### PR TITLE
Update the german localization !

### PR DESCRIPTION
**Pull request by nepomuk16321**

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -2722,49 +2722,49 @@
...
-      "de": null,
+      "de": "Stimme",
       "fr": null,
       "ru": "Голос",
       "pl": "Głos"
     },
     "Long Range Scanner": {
       "es": null,
-      "de": null,
+      "de": "Langstrecken-Scanner,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: missing double quote at the end

**Comment by nepomuk16321** [comment]
OK, I have corrected the text. Thanks for the hint. :-)

Greetings nepomuk

**Comment by msarilar** [comment]
thanks :)

**Comment by nepomuk16321** [comment]
No problem. We have to thank ! :-)

**Pull request merged**